src/nvim/normal_cmds.lua

```diff
@@ -0,0 +1,1158 @@
+local bit = require('bit')
+
+-- Description the NV_ flag values for cmd_flags
```

Comment:
What does this comment mean?

Reply:
emm before i am think also generate flag from script..so i leave this comment ..